nilearn/datasets/tests/test_struct.py

```diff
@@ -2,83 +2,20 @@
 
 # Authors: Alexandre Abraham, Ana Luisa Pinho
 
-import os
-import shutil
 from pathlib import Path
 
-import nibabel
 import numpy as np
 import pandas as pd
 import pytest
+from nibabel import Nifti1Image
 
-from nilearn.datasets import struct, utils
+from nilearn.datasets import struct
 from nilearn.datasets._testing import dict_to_archive, list_to_archive
 
 
-def test_get_dataset_dir(tmp_path):
```

Comment:
Same thing here.

Reply:
As you guessed this test was duplicated: see [here](https://github.com/nilearn/nilearn/issues/3570#issuecomment-1677562849)